packages/at_onboarding_cli/lib/src/onboard/at_onboarding_service_impl.dart

```diff
@@ -304,16 +344,18 @@ class AtOnboardingServiceImpl implements AtOnboardingService {
       Map<String, String> jsonData) async {
     String decryptionKey = _getDecryptionKey(jsonData);
     Map<String, String> atKeysMap = <String, String>{
-      AuthKeyType.pkamPublicKey: EncryptionUtil.decryptValue(
-          jsonData[AuthKeyType.pkamPublicKey]!, decryptionKey),
-      AuthKeyType.pkamPrivateKey: EncryptionUtil.decryptValue(
-          jsonData[AuthKeyType.pkamPrivateKey]!, decryptionKey),
       AuthKeyType.encryptionPublicKey: EncryptionUtil.decryptValue(
           jsonData[AuthKeyType.encryptionPublicKey]!, decryptionKey),
       AuthKeyType.encryptionPrivateKey: EncryptionUtil.decryptValue(
           jsonData[AuthKeyType.encryptionPrivateKey]!, decryptionKey),
       AuthKeyType.selfEncryptionKey: decryptionKey,
     };
+    if (atOnboardingPreference.authMode == PkamAuthMode.keysFile) {
```

Comment:
Need an explanation somewhere of why we are doing these special things when the authMode is PkamAuthMode.keysFile

Reply:
done